library/src/main/java/com/schibsted/spain/barista/action/BaristaCheckBoxActions.java

```diff
@@ -1,16 +0,0 @@
-package com.schibsted.spain.barista.action;
-
-import android.support.annotation.IdRes;
-
-import static com.schibsted.spain.barista.action.BaristaClickActions.clickOn;
-
-public class BaristaCheckBoxActions {
-
-  public static void clickCheckBoxItem(@IdRes int checkBoxId) {
```

Comment:
I'm not sure if deleting this method is savvy. As a Barista developer, you know that it's just a shortcut to a click, but as we offer lots of methods related to widgets, some one could search for `BaristaCheckBox` to find what he can do with checkboxes. Or it can also give more information about what's a given id. I worked on projects where you can't know what's an just looking at it, because some times we don't give great names, and having a `clickCheckBoxItem(R.id.list)` can give more information about what will be happening on the test. And... it's not too much code :·P

Reply:
Meeeeeeh I'm not convinced. You could give the same argument for other widgets like Button or Switch. And it's a new API to learn.

I could see the benefit of having dedicated methods for checking/unchecking a checkbox, but I can't see any benefit on having two different methods for the same thing 🤔

Reply:
If I can't convince you it means that it doesn't have sense. Let's remove those methods! 🎉 🎈 😂